main.py

```diff
@@ -0,0 +1,178 @@
+"""
+Main functionality for the smart home app.
+"""
+
+from python.database import AccessUserDatabase
+import hashlib
+
+
+class SmartHomeApp:
+    def __init__(self):
+        self.database = AccessUserDatabase()
+        self.logged_in_user_id = None
+
+    def hash_password(self, password):
+        """Hashes the password using SHA-256."""
+        hashed_password = hashlib.sha256(password.encode()).hexdigest()
+        return password
```

Comment:
```suggestion
        return hashed_password
```
returns the wrong variable.